package.json

```diff
@@ -1,3 +1,3 @@
 {
-  "version": "0.0.4"
+  "version": "0.0.6"
```

Comment:
remove this. it is automatically updated with the github action

Reply:
Removed. Sorry for that but i do not have experience with github actions since i use GitLab CI/CD.